src/java/src/test/java/ai/onnxruntime/genai/GenerationTest.java

```diff
@@ -25,79 +25,132 @@ public class GenerationTest {
   // phi-2 can be used in full end-to-end testing but needs to be manually downloaded.
   // it's also used this way in the C# unit tests.
   private static final String phi2ModelPath() {
-    String repoRoot = TestUtils.getRepoRoot();
-    File f = new File(repoRoot + "examples/python/example-models/phi2-int4-cpu");
-
-    if (!f.exists()) {
-      logger.warning("phi2 model not found at: " + f.getPath());
-      logger.warning(
-          "Please install as per https://github.com/microsoft/onnxruntime-genai/tree/rel-0.2.0/examples/csharp/HelloPhi2");
-      return null;
-    }
-
-    return f.getPath();
+    return TestUtils.getFilePathFromResource("/phi-2/int4/cpu");
   }
 
   @SuppressWarnings("unused") // Used in EnabledIf
   private static boolean havePhi2() {
     return phi2ModelPath() != null;
   }
 
+  @SuppressWarnings("unused") // Used in EnabledIf
+  private static boolean haveAdapters() {
+    return TestUtils.testAdapterTestModelPath() != null;
+  }
+
   @Test
   @EnabledIf("havePhi2")
   public void testUsageNoListener() throws GenAIException {
-    SimpleGenAI generator = new SimpleGenAI(phi2ModelPath());
-    GeneratorParams params = generator.createGeneratorParams("What's 6 times 7?");
-
-    String result = generator.generate(params, null);
-    logger.info("Result: " + result);
-    assertTrue(result.indexOf("Answer: 42") != -1);
```

Comment:
Are we not able to validate that the answer makes sense?

Reply:
Yes. Due to the phi-2 model being quite naive, the produced answer is garbage.

Reply:
>Is there potentially a tokenizer or config issue?

Not that I know of. I've checked with @baijumeswani to confirm this is OK. In fact, test cases in other language bindings exhibits the same behaviour of producing garbage output.